src/teehr/models/metrics/bootstrap_models.py

```diff
@@ -14,8 +15,8 @@
 
 # TODO: Extend an abstract base class for bootstrapping classes to ensure
 # that the necessary fields are included?  For Metrics too?
-class GumBootsModel(MetricsBasemodel):
-    """GumBoots bootstrapping.
+class GumbootModel(MetricsBasemodel):
+    """Gumboot bootstrapping.
 
     Parameters
```

Comment:
Need better documentation.  This doc string is the one a user is presented with when the type `Bootstrappers.Gumboot`.   This is a good place to tell the user how to use it.  Particularly what is required, what is not and what the default values are.